tiledb/sm/cpp_api/query.h

```diff
@@ -1086,6 +1086,79 @@ class Query {
         sizeof(char));
   }
 
+  /**
+   * Gets a buffer for a fixed-sized attribute.
+   *
+   * @param attr Attribute name
+   * @param buff Buffer array pointer with elements of the attribute type.
+   * @param nelements Number of array elements.
+   * @param element_size Size of array elements (in bytes).
+   **/
+  Query& get_buffer(
+      const std::string& attr,
```

Comment:
Let's use `name` to generically identify the attribute/dimension names. Same in the other function.